.circleci/continue_config.yml

```diff
@@ -437,6 +437,16 @@ workflows:
           enabled: true
           requires:
             - ci-base-<< matrix.python-version >>-<< matrix.extras >>
+      - integration-test:
+          name: integration-test-experimental-embedding-<< matrix.python-version >>-<< matrix.extras >>
```

Comment:
side question: did we not run this test before?

Reply:
No it was not run in CI before. My take is that it was [intentionally skipped as a todo](https://github.com/kolenaIO/kolena/pull/475/files#diff-8450575a09d4183f4c781d02b723d4c26fa3f3a313d9861f9bca4259e70e6e42L37) but lost track